pyatrea/__init__.py

```diff
@@ -146,16 +328,11 @@ def setPower(self, power):
             return False
         power -= 1
 
-        if(power < 99):
-            power = "0"+str(power)
-        elif(power < 10) and (power >= 0):
-            power = "00"+str(power)
-        elif(power == 100):
-            power = str(power)
-        else:
+        if(power < 0 or power > 100):
```

Comment:
While we are refactoring, can we change this to limit it to interval between 12 and 100? That is the one that my unit is accepting. Is this different to yours?

Reply:
Mine is the same, keeping to `power < 12`.